build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'fabric-loom' version '0.4-SNAPSHOT'
+    id 'fabric-loom' version '0.6-SNAPSHOT'
```

Comment:
If you are going to update loom, why not 0.8?

Reply:
I generally avoid the latest snapshot of Loom.